libnetdata/ebpf/ebpf.c

```diff
@@ -239,7 +239,8 @@ static int kernel_is_rejected()
     }
 
     fclose(kernel_reject_list);
-    freez(reject_string);
+    if (reject_string)
```

Comment:
isn't `free(NULL)` just a noop?

Reply:
You're right, I removed with last commit. thanks!